submission/loglist_manager.go

```diff
@@ -52,16 +52,16 @@ func NewLogListManager(llr LogListRefresher) *LogListManager {
 func (llm *LogListManager) Run(ctx context.Context, llRefresh time.Duration) {
 	llm.llRefreshInterval = llRefresh
 	go schedule.Every(ctx, llm.llRefreshInterval, func(ctx context.Context) {
```

Comment:
I don't think you need this wrapper func anymore. Just pass `llm.refreshLogListAndNotify` directly to `schedule.Every`.

Reply:
Done